src/sql/workbench/parts/dashboard/browser/widgets/properties/propertiesWidget.component.ts

```diff
@@ -25,15 +25,17 @@ export interface PropertiesConfig {
 
 export interface FlavorProperties {
 	flavor: string;
-	condition?: {
-		field: string;
-		operator: '==' | '<=' | '>=' | '!=';
-		value: string | boolean;
-	};
+	conditions?: Array<ConditionProperties>;
```

Comment:
This is a breaking change for any other extensions which are contributing these. I don't know of any (looks like PG SQL isn't) but we need to be careful here. @alanrenmsft @kevcunnane @kburtram Thoughts on this?

Reply:
On a side note we should also think about how do we want to introduce breaking changes to our extensions. What does vscode do when they need to break an API endpoint?

Reply:
Per offline chat - we are keeping both condition and conditions so we don't break any extension.